examples/rac-spectrum-tailwind/src/App.js

```diff
@@ -100,8 +109,9 @@ function SentimentRatingGroup() {
 function SentimentRating({ rating }) {
   return (
     <Radio
+      data-rac
```

Comment:
This `data-rac` won't be needed once released since we are adding this in attribute in `useRenderProps`.